system/Router/RouteCollection.php

```diff
@@ -1156,6 +1156,10 @@ public function environment(string $env, Closure $callback): RouteCollectionInte
      */
     public function reverseRoute(string $search, ...$params)
     {
+        if ($search === '') {
```

Comment:
Any reason to include `' '` as well?

Reply:
This is a safe guard. If `$search` is empty, the value will be `App\Controllers\`.
https://github.com/codeigniter4/CodeIgniter4/blob/52dbf70837526e9cdbd948248182006d17044f92/system/Router/RouteCollection.php#L1170-L1177
And it will match **the first route** for `App\Controllers`.